spec/controllers/collection_participants_controller_spec.rb

```diff
@@ -387,7 +392,8 @@
 
         it "approves those users, redirects to the collection participants page and displays a notification" do
           get :add, params
-          it_redirects_to collection_participants_path(collection)
+          expect(response).to have_http_status :redirect
```

Comment:
It wasn't I have updated it a bit and added the following comments.
          # We use it_redirects_to_internal as it_redirects_to
          # checks that there are no extra error,notice flash messages set
          # And in this case there are.